cl/users/email_handlers.py

```diff
@@ -454,3 +455,41 @@ def is_small_only_flagged(email_address: str) -> bool:
     if small_only.exists():
         return True
     return False
+
+
+def add_bcc_random(bcc_rate: float) -> bool:
+    """This function uses randint() to obtain the probability of getting a
+    number below a threshold defined by our bcc_rate which is a float value
+    from 0 to 1.
+
+    This function has a precision of 0.01, so we can determine the percentage
+    of messages that we want to BCC by a 1% precision.
+
+    The general idea is to get a random integer value between 1 to 100 and
+    compare it with our bcc_rate, firstly we need to normalize the bcc_rate
+    so we multiply it by 100, normalized_bcc_rate = bcc_rate * 100
+
+    We compare the returned value by the randint(1,100) method with the
+    following expression: returned_value <= normalized_bcc_rate
+
+    If the returned_value is equal to or below our normalized_bcc_rate, it
+    returns True, otherwise False.
+
+    So we can translate the bcc_rate to a probability.
+
+    e.g: if we want to bcc to the 10% of messages we should use a bcc_rate
+    of 0.1, so we'd have a normalized_bcc_rate of 10, so it'll check if the
+    random returned value by randint(1,100) is <= 10, which means, in theory,
+    a probability of 1/10.
```

Comment:
I don't think I've ever said this before, but this is too much detail. Do you think you can cut down? It's not a very complicated little function (even if we add the if/else I mentioned in my other comment).

Reply:
oh yes, it was too much, I've simplified it, I left just an example.